example/image-classification/symbols/alexnet.py

```diff
@@ -12,13 +12,13 @@ def get_symbol(num_classes, **kwargs):
         data=input_data, kernel=(11, 11), stride=(4, 4), num_filter=96)
     relu1 = mx.symbol.Activation(data=conv1, act_type="relu")
     pool1 = mx.symbol.Pooling(
-        data=relu1, pool_type="max", kernel=(3, 3), stride=(2,2))
+        data=relu1, pool_type="max", kernel=(3, 3), stride=(2,2), pad=(1, 1))
```

Comment:
yes.the pool1 no problem. no need for pad.
do other pool pad lead to compatibility issues?